python/MooseDocs/test/extensions/tests

```diff
@@ -47,4 +47,10 @@
     requirement = "The documentation system shall include tools for automatically retrieving application syntax."
     issues = '#9638'
   []
+  [alert]
+    type = PythonUnitTest
+    input = test_alert.py
+    requirement = "The documentation system shall include tools for creating alerts with markdown content wrapped in text boxes."
```

Comment:
```suggestion
    requirement = "The system shall include a documentation tool for creating alert text boxes."
```

Reply:
@aeslaughter Literally every other `requirement` input starts with `"The documentation system shall..."`, want me to remove the word "documentation" on all the others as well?

Reply:
Yes please. You could also do the following:

```
[Tests]
  [extensions]
    requirement = "The system shall include a documentation tool that includes"
    [alert]
       type = ...
      detail = "alert text boxes,"
   []
   [include]
     ...
     detail = "the ability to include content from a file,"
  []
[]
[]
```

Reply:
This is one requirement with many parts. There are lots of examples of this in moose/test and it is a feature that was added after these tests were created.